### ISSUE TITLE
<<a>> and [[a]] not supported.

### ISSUE THREAD

**Issue by cmal** [rasendubi/uniorg#77]
<<a>> and [[a]] not supported.

**Comment by ispringle** [comment]
Is `<<a>>` org syntax? As for `[[a]]` if should be working but you do have to implement your own link parsing for anything other than raw paths/URIs. So if your link is `[[file:/foo/bar]]` you need to implement the parsing of that to an actual file.

**Comment by rasendubi** [comment]
Yes, it's org syntax. <<a>> is a radio target and [[a]] is a radio link.
These are currently not implemented indeed

**Comment by ispringle** [comment]
Oh, I thought radio targets were a triplet of `<` `>` not a pair. 

https://orgmode.org/manual/Radio-Targets.html

Huh, this is called a "dedicated link". I'd not ever heard of that.

https://orgmode.org/manual/Internal-Links.html
